src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentPart/index.js

```diff
@@ -21,20 +36,21 @@ const IncidentPart = ({ index, attachments, subcategories, priorityList, splitFo
       sort
     />
     <FieldControlWrapper
+      data-testid="incidentPartFieldText"
       render={TextAreaInput}
       name={`part${index}.text`}
       display="Omschrijving"
       control={splitForm.get(`part${index}.text`)}
       rows={5}
     />
 
-    {attachments && attachments.length &&
+    {attachments && attachments.length ?
       <FieldControlWrapper
         render={CopyFileInput}
         name={`part${index}.image`}
         control={splitForm.get(`part${index}.image`)}
         values={attachments}
-      />
+      /> : ''
```

Comment:
If you don't want to render anything, the return value should be `null` and not an empty string

Reply:
gedaan